### DISCUSSION TITLE
Text formatting in rowy text fields

### DISCUSSION THREAD

**Discussion by Settld-Admin** [AntlerVC/firetable#733] [Support / Q&A] [answered]
Is is possible to add text formatting, in particular paragraph formatting, in rowy text fields?  Even if that is via markup or some other non GUI method?

**Comment by harinij** [comment, marked as answer]
@Settld-Admin We have two fields that can be useful for you here. One is Richtext field type and another is markdown. Please take a look these field types to see if it fits your usecase.

**Reply by Settld-Admin** [reply]
Thanks @harinij The rich text field type works great.

**Reply by Settld-Admin** [reply]
I'm afraid I spoke too soon.  In the Rowy UI changing the field type to Rich text worked well but unfortunately in Firestore the result of adding space between paragraph just results in <p> </p> tags being added around the text sections, which renders as is in the app UI. 
<img width="1358" alt="Screen Shot 2022-08-04 at 5 14 44 pm" src="https://user-images.githubusercontent.com/108853492/182787810-15978347-ab4a-4969-843a-041460ef0eff.png">
<img width="572" alt="Screen Shot 2022-08-04 at 5 15 23 pm" src="https://user-images.githubusercontent.com/108853492/182787835-31f83a89-786d-457a-ba87-1fde6f3e4341.png">

**Reply by Settld-Admin** [reply]
I have worked this out myself.  I just need to make adjustments in the app to read the html tags.
